### PR TITLE
(@wdio/cli): Simplified Electron setup

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

After releasing `v5.2.0` of our Electron service, we can simplify the setup required for running tests on Electron Applications.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Polish (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

This setup was tested on the [Electron Boilerplate](https://github.com/webdriverio/electron-boilerplate).

### CODE REVIEW

**Review by erwinheitzman — APPROVED** [review]
LGTM, fine improvements 👏

File: website/docs/desktop-testing/Electron.md [hunk 1]
```diff
@@ -87,10 +126,10 @@ ipcMain.handle('wdio-electron', () => {
...
-const someValue = await browser.electron.api('wow'); // default
-const someValue = await browser.electron.myCustomAPI('wow'); // configured using `customApiBrowserCommand`
+const someValue = await browser.electron.api('wow');
+console.log(someValue); // outputs: "such api"
 ```
 
 ## More Information
 
-You can learn more about how to configure the [`wdio-electron-service`](https://www.npmjs.com/package/wdio-electron-service) in the [service docs](/docs/wdio-electron-service).
+You can learn more about how to configure the [`wdio-electron-service`](https://www.npmjs.com/package/wdio-electron-service) in the [service docs](/docs/wdio-electron-service). Make sure to check out our [Electron boilerplate project](https://github.com/webdriverio/electron-boilerplate/actions/runs/6490587562/job/17626575914) that showcases how to intergate WebdriverIO in an example application.
```
Comment (goosewobbler) [inline website/docs/desktop-testing/Electron.md:135]: Do you mean to link to the CI output here?

File: website/docs/desktop-testing/Electron.md [hunk 2]
```diff
@@ -28,34 +28,73 @@ The configuration wizard will install all required packages and creates a `wdio.
...
 }
 ```
 
 And somewhere near the top of your main index file (app entry point):
 
 ```ts
+const isTest = process.env.NODE_ENV === 'test'
 if (isTest) {
-  import('wdio-electron-service/main');
+  require('wdio-electron-service/main');
```
Comment (goosewobbler) [inline website/docs/desktop-testing/Electron.md:80]: I noticed this change in the boilerplate project too, are there issues using dynamic import here?

File: website/docs/desktop-testing/Electron.md (hunk 1 above)
Comment (christian-bromann) [reply website/docs/desktop-testing/Electron.md:135]: Oh, good catch!

File: website/docs/desktop-testing/Electron.md (hunk 2 above)
Comment (christian-bromann) [reply website/docs/desktop-testing/Electron.md:80]: This I believe fails if your Electron project is forced to use a CJS setup in your `tsconfig.json`. AFAIK the Electron ecosystem still has no good support for ESM. While it is not impossible to get it working (looking at the service example), I wanted to make sure to not have it failing either due to type issues and using require here is just a safe way to demonstrate it.

File: website/docs/desktop-testing/Electron.md (hunk 1 above)
Comment (christian-bromann) [reply website/docs/desktop-testing/Electron.md:135]: fixed: https://github.com/webdriverio/webdriverio/commit/52cd25c36db38b47a17c9ff962c138ddf25b26ce

**Pull request merged**